validphys2/src/validphys/config.py

```diff
@@ -1716,7 +1743,9 @@ def produce_scale_variation_theories(self, theoryid, point_prescription):
         return {"theoryids": NSList(theoryids, nskey="theoryid")}
 
     @configparser.explicit_node
-    def produce_filter_data(self, fakedata: bool = False, theorycovmatconfig=None):
+    def produce_filter_data(
+        self, fakedata: bool = False, theorycovmatconfig=None, inconsistent_fakedata: bool = False
+    ):
```

Comment:
Please add it to the docstring.

Reply:
I guess a dict given the `.get` but just adding to the docstring what happens when `inconsistent_fakedata` is true is enough imho.